doc/psud/PSU_daemon_design.md

```diff
@@ -58,7 +61,10 @@ PSU information is stored in PSU table:
     voltage_max_threshold   = 1*3.3DIGIT                     ; the maximum voltage threshold of the PSU
     current                 = 1*3.3DIGIT                     ; the current of the PSU
     power                   = 1*3.3DIGIT                     ; the power of the PSU
-
+	  input_voltage           = 1*3.3DIGIT                     ; input voltage of the psu
+	  input_current           = 1*3.3DIGIT                     ; input current of the psu
+	  max_power               = 1*3.3DIGIT                     ; power capacity of the psu
```

Comment:
maybe `4.3DIGIT` for max power? I think it's possible for a PSU to provide a power greater than 1kw? how do you think?

Reply:
You right, done